cypress/e2e/transformation.cy.js

```diff
@@ -8,8 +8,8 @@ describe("Report transformation", () => {
     cy.get("[data-cy-debug='openSettings']").click();
     // Factory reset in settings dialog. Resets
     // transformation to factory value.
-    cy.get('button[title ^= "Reset and save factory settings"]').click();
-    cy.get("button[id=saveTableSettings]").click();
+    cy.get("[data-cy-settings='factorySettings']").click();
```

Comment:
```suggestion
    cy.get("[data-cy-settings='factoryReset']").click();
```

Reply:
Verwerkt. Het is niet nodig dat je bij het reviewen alle plekken opzoekt waar een wijziging voorkomt. Het is makkelijker voor mij om in VS Code de zoekfunctie te gebruiken. Dan weet ik zeker dat ik alles heb.